book/src/configuration.md

```diff
@@ -42,6 +42,7 @@ hidden = false
 | `completion-trigger-len` | The min-length of word under cursor to trigger autocompletion | `2` |
 | `auto-info` | Whether to display infoboxes | `true` |
 | `true-color` | Set to `true` to override automatic detection of terminal truecolor support in the event of a false negative. | `false` |
+| `color-column` | List of column positions at which to display the color columns. | `[]` |
```

Comment:
This sounds very useful but I personally really like having a global color column. It would be tedious to having to set it up for each language. So I tried to go with both. The language one having the priority but falling back to the editor one if not set.